packages/version/src/version-command.ts

```diff
@@ -477,7 +477,16 @@ export class VersionCommand extends Command<VersionCommandOption> {
       ? () => true
       : (_node, name) => prereleasePackageNames.has(name);
 
-    return collectPackages(this.packageGraph, { isCandidate }).map((pkg) => pkg?.name ?? '');
+    const prePkgs = collectPackages(this.packageGraph, { isCandidate }).map((pkg) => pkg?.name ?? '');
+
+    if (typeof this.options.conventionalPrerelease === 'string' && prePkgs.length === 0) {
```

Comment:
How about `typeof this.options.conventionalPrerelease !== 'boolean'`?

Reason:
`--conventionalPrerelease 0.2` will be a number and not a string.

Reply:
I'm not sure about "0.2", I'll need to check that one, but I did try type of boolean and that will cause a regression because the user can pass `*` (for all pkgs) and that is a string not a boolean

Reply:
> I'm not sure about "0.2", I'll need to check that one, but I did try type of boolean and that will cause a regression because the user can pass `*` (for all pkgs) and that is a string not a boolean

I had assumed that `*` was handled by `collectPackages`, in case it is not, then 
`typeof this.options.conventionalPrerelease !== 'boolean' && this.options.conventionalPrerelease !== '*'`

See: https://github.com/lerna-lite/lerna-lite/pull/576/files#diff-1c9ebfce0dc030f4173aa6f9f0dba5dd0651871de2d74e956d0f6e321fd1fa1fR26